ui/components/app/tab-bar/index.scss

```diff
@@ -68,6 +68,7 @@
         justify-content: center;
         margin-inline-end: 16px;
         flex: 0 0 18px;
+        color: var(--color-icon-alternative);
 
         @media screen and (min-width: $break-large) {
           flex: 0 0 14px;
```

Comment:
Updating settings tab menu icons to be less contrasting. Follows same pattern as account menu

<img width="365" alt="Screen Shot 2022-05-19 at 4 01 45 PM" src="https://user-images.githubusercontent.com/8112138/169418114-c10d3a4c-5864-47b5-a672-9b32d36494fd.png">